confignode/src/main/java/org/apache/iotdb/confignode/manager/load/LoadManager.java

```diff
@@ -327,7 +327,8 @@ private void updateNodeLoadStatistic() {
       broadcastLatestRegionRouteMap();
     }
     if (nodeCacheMap.size() == getNodeManager().getRegisteredNodeCount()) {
-      addMetrics();
+      addNodeMetrics();
+      addLeaderCount();
```

Comment:
UpdateNodeLoadStatistic method is executed once in a second, do `addNodeMetrics();` and `addLeaderCount();` need to be executed so frequently?

![image](https://user-images.githubusercontent.com/6756545/183584881-ef32465c-b765-47a2-bd36-baa150bc9601.png)